### PR TITLE
Added gobo presets in Clay-Paky, Contest, DTS, Dune_Ligting, Elation, Elumen8

### PR DESCRIPTION
**Pull request by kluszon**
Added gobo presets in Clay-Paky, Contest, DTS, Dune_Ligting, Elation, Elumen8

### CODE REVIEW

File: resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf [hunk 1]
```diff
@@ -151,28 +151,28 @@
...
  <Channel Name="Gobo 2 rotation">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="127">Fixed position</Capability>
-  <Capability Min="128" Max="190">Fast to slow rotation</Capability>
-  <Capability Min="191" Max="192">Stop rotation</Capability>
-  <Capability Min="193" Max="255">Slow to fast rotation</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Fixed position</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Fast to slow rotation</Capability>
+  <Capability Min="191" Max="192" Preset="RotationStop">Stop rotation</Capability>
+  <Capability Min="193" Max="255" Preset="RotationClockwiseSlowToFast">Slow to fast rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf:157]: Maybe in this row of four presets: RotationIndexed/RotationClockwiseFastToSlow/RotationStop the fourth one needs to be RotationCounterClockwiseSlowtoFast? like in channel Gobo 1 Rotation. If so, some presets below need to be changed, too.

File: resources/fixtures/Clay_Paky/Clay-Paky-Mini-Scan-HPE.qxf [hunk 2]
```diff
@@ -66,7 +60,7 @@
  </Channel>
  <Mode Name="Default">
   <Channel Number="0">Color Disc</Channel>
-  <Channel Number="1">Gobo Rotation</Channel>
+  <Channel Number="1" ActsOn="2">Gobo Rotation</Channel>
```
Comment (lady-darkness) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mini-Scan-HPE.qxf:63]: ActsOn="0"?

File: resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf [hunk 3]
```diff
@@ -220,13 +217,13 @@
...
-  <Channel Number="12">Gobo indexing</Channel>
+  <Channel Number="11" ActsOn="10">Gobo rotation</Channel>
+  <Channel Number="12" ActsOn="10">Gobo indexing</Channel>
   <Channel Number="13">Fixed Gobos</Channel>
-  <Channel Number="14">Fixed gobo indexing Fine</Channel>
+  <Channel Number="14" ActsOn="13">Fixed gobo indexing Fine</Channel>
   <Channel Number="15">Prism and gobo macros</Channel>
-  <Channel Number="16">Prism rotation</Channel>
-  <Channel Number="17">Prism Fine rotation</Channel>
+  <Channel Number="16" ActsOn="16">Prism rotation</Channel>
```
Comment (lady-darkness) [inline resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf:225]: ActsOn="15" ?

File: resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf [hunk 4]
```diff
@@ -220,13 +217,13 @@
...
+  <Channel Number="17" ActsOn="17">Prism Fine rotation</Channel>
```
Comment (lady-darkness) [inline resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf:226]: ActsOn="15" ?

File: resources/fixtures/Contest/Contest-EvoraB2R.qxf [hunk 5]
```diff
@@ -97,10 +97,10 @@
  </Channel>
  <Channel Name="Prism rotation">
   <Group Byte="0">Prism</Group>
-  <Capability Min="0" Max="127">Prism index</Capability>
-  <Capability Min="128" Max="189">Rotation CW F&gt;S</Capability>
-  <Capability Min="190" Max="193">Rotation stop</Capability>
-  <Capability Min="194" Max="255">Rotation CCW F&gt;S</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Prism index</Capability>
+  <Capability Min="128" Max="189" Preset="RotationCounterClockwiseFastToSlow">Rotation CW F&gt;S</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Contest/Contest-EvoraB2R.qxf:101]: Rotation Clockwise

File: resources/fixtures/DTS/DTS-Max-FAR.qxf [hunk 6]
```diff
@@ -173,10 +173,10 @@
  </Channel>
  <Channel Name="Gobo Rot/Index">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="9">Stop</Capability>
-  <Capability Min="10" Max="127">Left Rotation</Capability>
-  <Capability Min="128" Max="137">Stop</Capability>
-  <Capability Min="138" Max="255">Right Rotation</Capability>
+  <Capability Min="0" Max="9" Preset="RotationStop">Stop</Capability>
+  <Capability Min="10" Max="127" Preset="RotationClockwise">Left Rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-Max-FAR.qxf:177]: RotationCounterClockwise

File: resources/fixtures/DTS/DTS-Max-FAR.qxf [hunk 7]
```diff
@@ -173,10 +173,10 @@
...
+  <Capability Min="128" Max="137" Preset="RotationStop">Stop</Capability>
+  <Capability Min="138" Max="255" Preset="RotationCounterClockwise">Right Rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-Max-FAR.qxf:179]: RotationClockwise

File: resources/fixtures/DTS/DTS-Max-FAR.qxf [hunk 8]
```diff
@@ -273,21 +273,18 @@
...
+  <Capability Min="128" Max="255" Preset="PrismEffectOn" Res1="4">Prism</Capability>
  </Channel>
  <Channel Name="Prism Anim/Rot">
   <Group Byte="0">Prism</Group>
-  <Capability Min="0" Max="9">Stop</Capability>
-  <Capability Min="10" Max="127">Left Rotation</Capability>
-  <Capability Min="128" Max="137">Stop</Capability>
-  <Capability Min="138" Max="255">Right Rotation</Capability>
+  <Capability Min="0" Max="9" Preset="RotationStop">Stop</Capability>
+  <Capability Min="10" Max="127" Preset="RotationCounterClockwiseSlowToFast">Left Rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-Max-FAR.qxf:282]: without Speed? RotationCounterClockwise

File: resources/fixtures/DTS/DTS-Max-FAR.qxf [hunk 9]
```diff
@@ -273,21 +273,18 @@
...
+  <Capability Min="128" Max="137" Preset="RotationStop">Stop</Capability>
+  <Capability Min="138" Max="255" Preset="RotationClockwiseSlowToFast">Right Rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-Max-FAR.qxf:284]: RotationClockwise

File: resources/fixtures/DTS/DTS-XR7-Spot.qxf [hunk 10]
```diff
@@ -46,47 +42,31 @@
...
+  <Capability Min="99" Max="109" Preset="ColorDoubleMacro">Bicolour 5/6</Capability>
+  <Capability Min="110" Max="120" Preset="ColorMacro">Colour6</Capability>
+  <Capability Min="121" Max="131" Preset="ColorDoubleMacro">Bicolour 6/7</Capability>
+  <Capability Min="132" Max="142" Preset="ColorMacro">Colour7</Capability>
+  <Capability Min="143" Max="153" Preset="ColorDoubleMacro">Bicolour 7/8</Capability>
+  <Capability Min="154" Max="164" Preset="ColorMacro">Colour8</Capability>
+  <Capability Min="165" Max="175" Preset="ColorDoubleMacro">Bicolour 8/9</Capability>
+  <Capability Min="176" Max="186" Preset="ColorMacro">Colour9</Capability>
+  <Capability Min="187" Max="197" Preset="ColorDoubleMacro">Bicolour 9/1</Capability>
+  <Capability Min="198" Max="224" Preset="RotationClockwise">Right rotation speed min max</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-XR7-Spot.qxf:67]: RotationClockwiseSlowToFast

File: resources/fixtures/DTS/DTS-XR7-Spot.qxf [hunk 11]
```diff
@@ -46,47 +42,31 @@
...
+  <Capability Min="225" Max="228" Preset="RotationStop">Stop</Capability>
+  <Capability Min="229" Max="255" Preset="RotationCounterClockwise">Left rotation speed min max</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/DTS/DTS-XR7-Spot.qxf:69]: RotationCounterClockwiseSlowToFast

File: resources/fixtures/Elation/Elation-Platinum-Spot-III.qxf [hunk 12]
```diff
@@ -76,9 +76,9 @@
   <Capability Min="151" Max="163" Preset="GoboShakeMacro" Res1="">shake fixed gobo 5</Capability>
   <Capability Min="164" Max="176" Preset="GoboShakeMacro" Res1="">shake fixed gobo 6</Capability>
   <Capability Min="177" Max="189" Preset="GoboShakeMacro" Res1="Others/gobo00008.svg">shake fixed gobo 7</Capability>
-  <Capability Min="190" Max="221">clockwise gobo rotation</Capability>
-  <Capability Min="222" Max="223">no rotation</Capability>
-  <Capability Min="224" Max="255">conterclockwise gobo rotation</Capability>
+  <Capability Min="190" Max="221" Preset="RotationClockwiseFastToSlow">clockwise gobo rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Elation/Elation-Platinum-Spot-III.qxf:79]: RotationClockwise

File: resources/fixtures/Elation/Elation-Platinum-Spot-III.qxf [hunk 13]
```diff
@@ -76,9 +76,9 @@
...
+  <Capability Min="222" Max="223" Preset="RotationStop">no rotation</Capability>
+  <Capability Min="224" Max="255" Preset="RotationCounterClockwiseSlowToFast">conterclockwise gobo rotation</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Elation/Elation-Platinum-Spot-III.qxf:81]: RotationCounterClockwise

File: resources/fixtures/Elation/Elation-Proteus-Beam.qxf [hunk 14]
```diff
@@ -170,35 +170,35 @@
...
-  <Capability Min="184" Max="191">Macro 20</Capability>
-  <Capability Min="192" Max="199">Macro 21</Capability>
-  <Capability Min="200" Max="207">Macro 22</Capability>
-  <Capability Min="208" Max="215">Macro 23</Capability>
-  <Capability Min="216" Max="223">Macro 24</Capability>
-  <Capability Min="224" Max="231">Macro 25</Capability>
-  <Capability Min="232" Max="239">Macro 26</Capability>
-  <Capability Min="240" Max="247">Macro 27</Capability>
-  <Capability Min="248" Max="255">Random CMY</Capability>
+  <Capability Min="0" Max="31" Preset="ColorMacro">Off</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Elation/Elation-Proteus-Beam.qxf:173]: "ColorMacro" Res1="#ffffff" ?

File: resources/fixtures/Elumen8/Elumen8-MS-700PE.qxf [hunk 15]
```diff
@@ -64,16 +64,16 @@
...
  <Channel Name="Gobo 1 Rotation">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="127">Indexed Rotation</Capability>
-  <Capability Min="128" Max="189">CW Fast - Slow</Capability>
-  <Capability Min="190" Max="193">Stop</Capability>
-  <Capability Min="194" Max="255">CCW Slow - Fast</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Indexed Rotation</Capability>
+  <Capability Min="128" Max="189" Preset="RotationClockwiseFastToSlow">CW Fast - Slow</Capability>
+  <Capability Min="190" Max="193" Preset="RotationStop">Stop</Capability>
+  <Capability Min="194" Max="255" Preset="RotationClockwiseSlowToFast">CCW Slow - Fast</Capability>
```
Comment (lady-darkness) [inline resources/fixtures/Elumen8/Elumen8-MS-700PE.qxf:76]: RotationCounterClockwiseSlowToFast

File: resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf (hunk 1 above)
Comment (joshuaurieli) [reply resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf:157]: I checked the manual and you are correct. @kluszon please change the preset on line **155** to RotationCounterClockwiseFastToSlow

File: resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf [hunk 16]
```diff
@@ -151,28 +151,28 @@
...
  </Channel>
  <Channel Name="Prism rotation">
   <Group Byte="0">Effect</Group>
-  <Capability Min="0" Max="127">Fixed position</Capability>
-  <Capability Min="128" Max="190">Fast to slow rotation</Capability>
-  <Capability Min="191" Max="192">Stop rotation</Capability>
-  <Capability Min="193" Max="255">Slow to fast rotation</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Fixed position</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Fast to slow rotation</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf:162]: @kluszon please change the preset on line 162 to RotationCounterClockwiseFastToSlow

File: resources/fixtures/Clay_Paky/Clay-Paky-Mini-Scan-HPE.qxf (hunk 2 above)
Comment (joshuaurieli) [reply resources/fixtures/Clay_Paky/Clay-Paky-Mini-Scan-HPE.qxf:63]: ActsOn="2" is correct, as the gobo selection here is on channel 2 (it's a strange device)

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 17]
```diff
@@ -75,9 +75,9 @@
   <Capability Min="61" Max="64" Preset="GoboMacro" Res1="ClayPaky/gobo00079.svg">Gobo 16</Capability>
   <Capability Min="65" Max="68" Preset="GoboMacro" Res1="ClayPaky/gobo00028.svg">Gobo 17</Capability>
   <Capability Min="69" Max="71" Preset="GoboMacro" Res1="SGM/gobo00001.svg">Gobo 18</Capability>
-  <Capability Min="72" Max="113">Gobo rotation Fast&gt;Slow</Capability>
-  <Capability Min="114" Max="117">Stop Rotation</Capability>
-  <Capability Min="118" Max="159">Gobo rotation Slow&gt;Fast</Capability>
+  <Capability Min="72" Max="113" Preset="RotationClockwiseFastToSlow">Gobo rotation Fast&gt;Slow</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:78]: @kluszon Preset="RotationCounterClockwiseFastToSlow"

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 18]
```diff
@@ -75,9 +75,9 @@
...
+  <Capability Min="114" Max="117" Preset="RotationStop">Stop Rotation</Capability>
+  <Capability Min="118" Max="159" Preset="RotationCounterClockwiseSlowToFast">Gobo rotation Slow&gt;Fast</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:80]: @kluszon Preset="RotationClockwiseSlowToFast"

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 19]
```diff
@@ -103,13 +103,13 @@
  </Channel>
  <Channel Name="Disk Rotation">
   <Group Byte="0">Effect</Group>
-  <Capability Min="0" Max="124">Disk Rotation Fast&gt;Slow</Capability>
-  <Capability Min="125" Max="130">Stop</Capability>
-  <Capability Min="131" Max="255">Disk Rotation Slow&lt;Fast</Capability>
+  <Capability Min="0" Max="124" Preset="RotationClockwiseFastToSlow">Disk Rotation Fast&gt;Slow</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:106]: @kluszon Preset="RotationCounterClockwiseFastToSlow"

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 20]
```diff
@@ -103,13 +103,13 @@
...
+  <Capability Min="125" Max="130" Preset="RotationStop">Stop</Capability>
+  <Capability Min="131" Max="255" Preset="RotationCounterClockwiseSlowToFast">Disk Rotation Slow&lt;Fast</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:108]: @kluszon Preset="RotationClockwiseSlowToFast"

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 21]
```diff
@@ -125,34 +125,24 @@
...
-  <Capability Min="21" Max="41">Prism Index 90° to 180°</Capability>
-  <Capability Min="42" Max="62">Prism Index 180° to 270°</Capability>
-  <Capability Min="63" Max="83">Prism Index 270° to 360°</Capability>
-  <Capability Min="84" Max="104">Prism Index 360° to 450°</Capability>
-  <Capability Min="105" Max="127">Prism Index 450° to 540°</Capability>
-  <Capability Min="128" Max="190">Rotation Max &lt; Min</Capability>
-  <Capability Min="191" Max="192">Stop Rotation</Capability>
-  <Capability Min="193" Max="255">Rotation Min &lt; Max</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Rotation Indexed</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Rotation Max &lt; Min</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:143]: @kluszon Preset="RotationCounterClockwiseFastToSlow"

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 22]
```diff
@@ -252,12 +242,12 @@
   <Channel Number="8">Dimmer Fine</Channel>
   <Channel Number="9">Static Gobo</Channel>
   <Channel Number="10">Disk Insert</Channel>
-  <Channel Number="11">Disk Rotation</Channel>
+  <Channel Number="11" ActsOn="10">Disk Rotation</Channel>
   <Channel Number="12">Rotation Gobo</Channel>
-  <Channel Number="13">Gobo Rotation</Channel>
+  <Channel Number="13" ActsOn="12">Gobo Rotation</Channel>
   <Channel Number="14">Gobo Rotation Fine</Channel>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:248]: @kluszon please add ActsOn="13

File: resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf [hunk 23]
```diff
@@ -284,12 +274,12 @@
...
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Mythos.qxf:280]: @kluszon please add ActsOn="13

File: resources/fixtures/Clay_Paky/Clay-Paky-Sharpy.qxf [hunk 24]
```diff
@@ -109,9 +109,9 @@
   <Capability Min="84" Max="104">Position 360°</Capability>
   <Capability Min="105" Max="126">Position 450°</Capability>
   <Capability Min="127" Max="127">Position 540°</Capability>
-  <Capability Min="128" Max="190">Rotation (Fast 43rpm to slow 1.1rph)</Capability>
-  <Capability Min="191" Max="192">Stop</Capability>
-  <Capability Min="193" Max="255">Rotation (Slow 1.1rph to fast 43rpm)</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Rotation (Fast 43rpm to slow 1.1rph)</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Sharpy.qxf:112]: @kluszon Preset="RotationCounterClockwiseFastToSlow"

File: resources/fixtures/Clay_Paky/Clay-Paky-Sharpy.qxf [hunk 25]
```diff
@@ -109,9 +109,9 @@
...
+  <Capability Min="191" Max="192" Preset="RotationStop">Stop</Capability>
+  <Capability Min="193" Max="255" Preset="RotationCounterClockwiseSlowToFast">Rotation (Slow 1.1rph to fast 43rpm)</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-Sharpy.qxf:114]: @kluszon Preset="RotationClockwiseSlowToFast"

File: resources/fixtures/Clay_Paky/Clay-Paky-SuperSharpy.qxf [hunk 26]
```diff
@@ -154,9 +154,9 @@
   <Capability Min="63" Max="83">Prism Index 270° to 360°</Capability>
   <Capability Min="84" Max="104">Prism Index 360° to 450°</Capability>
   <Capability Min="105" Max="127">Prism Index 450° to 540°</Capability>
-  <Capability Min="128" Max="190">Rotation Max &lt; Min</Capability>
-  <Capability Min="191" Max="192">Stop Rotation</Capability>
-  <Capability Min="193" Max="255">Rotation Min &lt; Max</Capability>
+  <Capability Min="128" Max="190" Preset="RotationClockwiseFastToSlow">Rotation Max &lt; Min</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-SuperSharpy.qxf:157]: @kluszon Preset="RotationCounterClockwiseFastToSlow"

File: resources/fixtures/Clay_Paky/Clay-Paky-SuperSharpy.qxf [hunk 27]
```diff
@@ -154,9 +154,9 @@
...
+  <Capability Min="191" Max="192" Preset="RotationStop">Stop Rotation</Capability>
+  <Capability Min="193" Max="255" Preset="RotationCounterClockwiseSlowToFast">Rotation Min &lt; Max</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Clay_Paky/Clay-Paky-SuperSharpy.qxf:159]: @kluszon Preset="RotationClockwiseSlowToFast"

File: resources/fixtures/Color_Imagination/Color-Imagination-LedSpot-90-SI-052.qxf [hunk 28]
```diff
@@ -23,58 +23,58 @@
   <Capability Min="80" Max="95">Color 5</Capability>
   <Capability Min="96" Max="111">Color 6</Capability>
   <Capability Min="112" Max="127">Color 7</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Color_Imagination/Color-Imagination-LedSpot-90-SI-052.qxf:25]: @kluszon Please add Preset="ColorMacro" to all relevant lines above

File: resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf (hunk 3 above)
Comment (joshuaurieli) [reply resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf:225]: @kluszon Please correct

File: resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf (hunk 4 above)
Comment (joshuaurieli) [reply resources/fixtures/Contest/Contest-Evora-Beam-15R.qxf:226]: @kluszon Please correct

File: resources/fixtures/Contest/Contest-Evora-DUO-B2R.qxf [hunk 29]
```diff
@@ -42,108 +42,108 @@
...
-  <Capability Min="117" Max="127">Colour 13</Capability>
-  <Capability Min="128" Max="189">Clockwise, fast to slow Rainbow effect</Capability>
-  <Capability Min="190" Max="193">No rotation</Capability>
-  <Capability Min="194" Max="255">Counterclockwise, slow to fast Rainbow effect</Capability>
+  <Capability Min="9" Max="17" Preset="ColorMacro">Colour 1</Capability>
+  <Capability Min="18" Max="26" Preset="ColorMacro">Colour 2</Capability>
+  <Capability Min="27" Max="35" Preset="ColorMacro">Colour 3</Capability>
+  <Capability Min="36" Max="44" Preset="ColorDoubleMacro">Colour 4</Capability>
+  <Capability Min="45" Max="53" Preset="ColorDoubleMacro">Colour 5</Capability>
+  <Capability Min="54" Max="62" Preset="ColorMacro">Colour 6</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Contest/Contest-Evora-DUO-B2R.qxf:70]: @kluszon Please correct to Preset="ColorMacro"

File: resources/fixtures/Contest/Contest-EvoraB2R.qxf (hunk 5 above)
Comment (joshuaurieli) [reply resources/fixtures/Contest/Contest-EvoraB2R.qxf:101]: @kluszon please correct

File: resources/fixtures/DTS/DTS-Max-FAR.qxf (hunk 6 above)
Comment (joshuaurieli) [reply resources/fixtures/DTS/DTS-Max-FAR.qxf:177]: @kluszon Please correct

File: resources/fixtures/DTS/DTS-Max-FAR.qxf (hunk 7 above)
Comment (joshuaurieli) [reply resources/fixtures/DTS/DTS-Max-FAR.qxf:179]: @kluszon Please correct

File: resources/fixtures/DTS/DTS-Max-FAR.qxf (hunk 8 above)
Comment (joshuaurieli) [reply resources/fixtures/DTS/DTS-Max-FAR.qxf:282]: We should leave speed

File: resources/fixtures/DTS/DTS-Max-FAR.qxf (hunk 9 above)
Comment (joshuaurieli) [reply resources/fixtures/DTS/DTS-Max-FAR.qxf:284]: We should have speed

**Pull request closed without merging**